yarn/src/main/scala/org/apache/spark/deploy/yarn/Client.scala

```diff
@@ -1337,55 +1337,9 @@ object Client extends Logging {
       conf: Configuration,
       credentials: Credentials) {
     if (shouldGetTokens(sparkConf, "hive") && UserGroupInformation.isSecurityEnabled) {
-      val mirror = universe.runtimeMirror(getClass.getClassLoader)
```

Comment:
this should go to `utils.getContextOrSparkClassLoader()`; notable that scalastyle doesn't pick up on this, even though it rejects `Class.forName()` since SPARK-8962